### PR TITLE
updating next-gen docs

### PR DESCRIPTION
**Pull request by slab14**
Updates:
1.  To allow build on Ubuntu VM, removing USER docker from Dockerfile.
2. adding comment about tool size to documentation 
3. adding check to see if about to remove existing directory (unable to add user option to continue kept receiving errors when running nested bash if loops inside Makefile). 
4. Minor change to current-gen documentation to match version in text to version in code listing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -103,6 +103,14 @@ frontend: build_bootstrap
 install: build_bootstrap
 	$(call docker_run,make -f install.mk, -w all)
 	@echo Populating namespace within: $(USPARK_NAMESPACEROOTDIR)...
+	@if [ -d $(USPARK_NAMESPACEROOTDIR) ]; then \
+		echo "$(USPARK_NAMESPACEROOTDIR) already exists. "; \
+		read -p "Would you like to continue (rm -rf $(USPARK_NAMESPACEROOTDIR)) [y/N]? " action; \
+		if [ "$$action" = "y" ] && [ "$$action" = "Y" ]; then \
+			echo "Please remove $(USPARK_NAMESPACEROOTDIR) (e.g., rm -rf $(USPARK_NAMESPACEROOTDIR)) in order to allow installation"; \
+			exit 1; \
+		fi; \
```
Comment (amitvasudevan) [inline Makefile]: This should $$action= "n" && $$action = "N". In that case, print out the message you have there and exit. IF the user says "Y/y" then we go ahead and remove the directory and move on with the installation.

File: build-trusses/build-x86_64.Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ RUN sudo apk update &&\
     sudo chown root:root /etc/sudoers.d/docker &&\
     sudo sed -i.bak 's/^Defaults.*requiretty//g' /etc/sudoers
 
-USER docker
+#USER docker
```
Comment (amitvasudevan) [inline build-trusses/build-x86_64.Dockerfile]: You may want to add a comment that this is commented out currently due to UID conflicts with alpine distros

File: Makefile (hunk 1 above)
Comment (slab14) [reply Makefile]: Yes, changed to $$action != "y"/"Y".

File: build-trusses/build-x86_64.Dockerfile (hunk 2 above)
Comment (slab14) [reply build-trusses/build-x86_64.Dockerfile]: added comment

**Pull request merged**